Packs/IBMResilientSystems/Scripts/IbmConvertCommentsToTable/README.md

```diff
@@ -0,0 +1,21 @@
+This script is used to format IBM QRadar SOAR notes into a mark-down table.
```

Comment:
Should this be comments? (and it should be markdown not mark-down)
This script is used to convert IBM QRadar SOAR comments to a markdown table.